retrieval/resource_map.go

```diff
@@ -117,10 +117,25 @@ var ProxyResourceMap = ResourceMap{
 	},
 }
 
+var AggregatedProxyResourceMap = ResourceMap{
+	Type:       "apigee.googleapis.com/ProxyV2",
+	MatchLabel: "__meta_kubernetes_pod_label_type_aggregate_proxy",
```

Comment:
Is there any prior agreement that the pod label is `aggregate_proxy`? If yes, I am fine with it;

but if not, I think the pod_label as `proxy_v2` and the match label as `__meta_kubernetes_pod_label_type_proxy_v2` may be reasonable from the user perspective, not from how it is implemented.

What do you think?

Reply:
Sounds Good. Updated the CL with the same.